README.md

```diff
@@ -152,6 +152,7 @@ Use the "Table of Contents" menu on the top-left corner to explore the list.
 - [vite-react-starter](https://github.com/fabgrel10/vite-react-starter) - React + ESLint + Prettier + SASS + Emotion + TailwindCSS + Jest + Testing Library.
 - [chrome-extension-boilerplate-react-vite](https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite) - React + TypeScript + SASS + Prettier + ESLint + GitHub Actions + Chrome Extension Manifest v3.
 - [vite-ant-design-pro](https://github.com/1247748612/vite-ant-design-pro) - React + antd + ProComponents + ReactQuery + ReactRouter6.
+- [React-PWA](https://github.com/suren-atoyan/react-pwa) - Starter kit for modern web applications.
```

Comment:
Describe the key technologies that were used, the description is too vague.

Reply:
could you please check it? it's updated